tests/engine/test_launch.py

```diff
@@ -141,6 +141,24 @@ def test_submit_store_provenance_false(self):
         with self.assertRaises(exceptions.InvalidOperation):
             launch.submit(AddWorkChain, term_a=self.term_a, term_b=self.term_b, metadata={'store_provenance': False})
 
+    def test_submit_process_function(self):
```

Comment:
Since this was supposed to test submitting a process function, but it not actually does (and this is done isntead in `test_daemon.py`, this entire test can be removed.